pkg/crc/machine/start.go

```diff
@@ -476,6 +479,31 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 		logging.Warn(fmt.Sprintf("Failed to query DNS from host: %v", err))
 	}
 
+	if vm.bundle.IsMicroshift() {
```

Comment:
Fwiw, the more preset checks such as this one, the messier our code becomes. Especially in the already far too big `Start` method.
One possibel alternative is to try to add interfaces (here, could be a `Starter` interface, with a `PostVMStart` method maybe?), and have a `PodmanStarter`, `OpenShiftStarter`, ... which would all implement this interface.
This is a bit artificial here, especially as I haven't spent much time thinking about it, but it might apply better in other parts of the code.